connector/src/main/java/com/microsoft/kusto/spark/javaUtil/BufferWriterWithProgress.java

```diff
@@ -0,0 +1,289 @@
+package com.microsoft.kusto.spark.javaUtil;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.*;
+
+/*
+ * Copyright (c) 1996, 2013, Oracle and/or its affiliates. All rights reserved.
```

Comment:
where did you copy this from? what are the license terms?

Reply:
its from java.io.BufferWriter class - just added the progress thing

Reply:
link?

Reply:
to the code itself ? took it from intellij .. but its just this class : https://docs.oracle.com/javase/7/docs/api/java/io/BufferedWriter.html
how am i suppose to do it right ? write it myself would be better ?